public/js/services/AtomsApi.js

```diff
@@ -4,44 +4,46 @@ export default {
 
   getAtom: (atomType, atomId) => {
     return pandaFetch(
-      `/api/preview/${atomType}/${atomId}`,
-      {
-        method: 'get',
-        credentials: 'same-origin'
-      }
+      new Request(
```

Comment:
These didn't seem to work when I changed polyfills before, so I was just sending them separately. `pandaFetch.js` may need it's params changing.